src/live-preview.ts

```diff
@@ -382,9 +438,20 @@ export default class LivePreview {
             this.config.editButton.enable
         ) {
             const tooltip = document.createElement("button");
+            const tooltipInnerContainer = document.createElement("div");
+            tooltipInnerContainer.classList.add("cslp-tooltip-inner-container");
+            tooltip.classList.add("cslp-tooltip");
```

Comment:
Why do we need this element?

Reply:
This extra element ensures that the edit button does not hide until the customer has completely moved away from the currently hovered element and its associated edit button, without impacting the overall look of the button.